src/features/home/components/modules/VideoModule.tsx

```diff
@@ -46,7 +45,10 @@ export const VideoModule: FunctionComponent<VideoModuleProps> = (props) => {
       <VideoOfferContainer>
         <Typo.Title3>{props.title}</Typo.Title3>
         <Spacer.Column numberOfSpaces={5} />
-        <StyledTouchable onPress={showVideoModal} testID={'video-thumbnail'}>
+        <StyledTouchableHighlight
+          onPress={showVideoModal}
+          testID={'video-thumbnail'}
```

Comment:
```suggestion
          testID="video-thumbnail"
          accessibilityRole="button"
```
Avant on utilisait Touchable qui avait déjà le `role` "button", mais maintenant qu'on utilise `TouchableHighlight` il faut le préciser.